docs/operations/rule-configuration.md

```diff
@@ -28,7 +28,10 @@ In Apache Druid (incubating), Coordinator processes use rules to determine what
 There are three types of rules, i.e., load rules, drop rules, and broadcast rules. Load rules indicate how segments should be assigned to different historical process tiers and how many replicas of a segment should exist in each tier.
 Drop rules indicate when segments should be dropped entirely from the cluster. Finally, broadcast rules indicate how segments of different data sources should be co-located in Historical processes.
 
-The Coordinator loads a set of rules from the metadata storage. Rules may be specific to a certain datasource and/or a default set of rules can be configured. Rules are read in order and hence the ordering of rules is important. The Coordinator will cycle through all available segments and match each segment with the first rule that applies. Each segment may only match a single rule.
+The Coordinator loads a set of rules from the metadata storage. Rules may be specific to a certain data source and/or a
```

Comment:
There is a separate issue for this: https://github.com/apache/druid/issues/8917

Reply:
Thanks for bringing up #8917. That issue looks to use only datasource instead of data source. Do we need this change even though the original spelling here was datasource?

Reply:
Replaced all occurrences of "data source" with "datasource" in this particular doc.